### PR TITLE
formatting and cleanup

### PR DESCRIPTION
**Pull request by muresanrazvan**
formatted all files and cleaned up unnecessary imports/comments/casts/constructors

### CODE REVIEW

**Review by mihaiparvu — APPROVED** [review]

File: src/spyGui/SpyServer.java [hunk 1]
```diff
@@ -9,39 +9,37 @@
...
-	}
+    public void run() {
+        try {
+            int port = 0;
+            ServerSocket ss = new ServerSocket(port);
+            port = ss.getLocalPort();
+            writePortFile(port);
+            System.out.println("Server started...");
+            while (true) {
+
```
Comment (rticau) [inline src/spyGui/SpyServer.java]: remove empty line

File: src/spyGui/SpyFrame.java [hunk 2]
```diff
@@ -20,82 +20,73 @@
...
+                    aboutDial.setAlwaysOnTop(true);
+                    aboutDial.pack();
+                    aboutDial.setVisible(true);
+                } else if (arg0.getActionCommand().equals("Launch")) {
+                    launchDlg.setLocation(frame.getX() + 10, frame.getY() + 200);
+                    launchDlg.setAlwaysOnTop(true);
+                    launchDlg.pack();
+                    launchDlg.setVisible(true);
+                }
+
```
Comment (rticau) [inline src/spyGui/SpyFrame.java:69]: two empty lines

File: src/spyAgent/CompMouseListner.java [hunk 3]
```diff
@@ -15,87 +15,83 @@
...
-
-	public void mouseClicked(MouseEvent arg0) {
-		Component comp=arg0.getComponent();
-		if(comp instanceof JTabbedPane){
-			JTabbedPane tabpane=(JTabbedPane)comp;
-			int tabIndex=tabpane.getSelectedIndex();
-			String tabName=tabpane.getTitleAt(tabIndex);
-			String tabText=",Tab Index - "+tabIndex+",Tab Title- "+tabName;
-			Communicator.writeToServer("Window Title - "+winTitle+",Index - "+myIndex+",Instance Of - "+classType+",Comp. Hierarchy - "+compHierarchy+","+tabpane.toString()+tabText);
-			//ClientLogger.Logger("Index="+myIndex+","+comp.toString());
```
Comment (rticau) [inline src/spyAgent/CompMouseListner.java:48]: we don't need this log in the future? for debugging maybe?

Comment (muresanrazvan) [reply src/spyAgent/CompMouseListner.java:48]: no, we can write our own logs if needed.

**Review by rticau — APPROVED** [review]

**Review by andreeakovacs — APPROVED** [review]

**Review by oanab11 — APPROVED** [review]

**Pull request merged**